users/users.md

```diff
@@ -117,6 +118,80 @@ Response
 ```
 ---
 
+## Create user account
+
+Create a user account.
+
+```apacheconfig
+POST /v1/users
+```
+
+#### Path parameters
+The path parameters are not expected.
+
+#### Query parameters
+The query parameters are not expected.
+
+#### Request body
+
+| Name | Type | Description | Required |
```

Comment:
I would add an extra column with an example as well.

Reply:
The json example is shown underneath the table. Do we still need it in the table?

Reply:
I would do that for readability. Someone may want to refer to the table, someone would just grab the json when the read the doc.